### PR TITLE
Replace internal links between icm documents with relative links

### LINKED ISSUE #187: Replace internal links between ICM documents with relative links to stay within the same release tree / fork / release package
**Problem description**

documentation/CAMARA-API-access-and-user-consent.md contains about six links to the Profile document which are written as fully qualified URLs which are pointing to the main branch of ICM. The Profile contains one such link which points to [documentation/CAMARA-API-access-and-user-consent.md](https://github.com/camaraproject/IdentityAndConsentManagement/blob/main/documentation/CAMARA-API-access-and-user-consent.md)

A reader within a release tree would jump out of the tree into a potential wrong version. The same for someone who has downloaded the release package (zip-file) of ICM.

**Expected behavior**

Use - as [recommend](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#relative-links) by GitHub - relative links instead, concrete with the following replacements:
`https://github.com/camaraproject/IdentityAndConsentManagement/blob/main/documentation/CAMARA-Security-Interoperability.md` to be replace with `CAMARA-Security-Interoperability.md`
`https://github.com/camaraproject/IdentityAndConsentManagement/blob/main/documentation/CAMARA-API-access-and-user-consent.md` to be replaced with `CAMARA-API-access-and-user-consent.md`

**Alternative solution**

**Additional context**

### PR DESCRIPTION
**Pull request by hdamker**
#### What type of PR is this?

Add one of the following kinds:
* correction

#### What this PR does / why we need it:

See #187 

#### Which issue(s) this PR fixes:

Fixes #187 

#### Special notes for reviewers:

Tested on GitHub within the PR source branch and with local renderer (Xcode). 

#### Changelog input

```
 Replaced internal links between ICM documents with relative links
```

### CODE REVIEW

File: documentation/CAMARA-API-access-and-user-consent.md [hunk 1]
```diff
@@ -136,7 +136,7 @@ Note over BE,FE: Response
...
 
-As per the standard authorization code flow, the device application is redirected to the operator authorization endpoint in API exposure platform (Steps 1-2), providing a redirect_uri (invoker_callback) pointing to the invoker backend (where the auth code will eventually be sent), as well as the purpose for accessing the data. NOTE: The way to declare a purpose when accessing the CAMARA APIs is defined in [the CAMARA Security and Interoperability Profile](https://github.com/camaraproject/IdentityAndConsentManagement/blob/main/documentation/CAMARA-Security-Interoperability.md#purpose-as-a-scope).
+As per the standard authorization code flow, the device application is redirected to the operator authorization endpoint in API exposure platform (Steps 1-2), providing a redirect_uri (invoker_callback) pointing to the invoker backend (where the auth code will eventually be sent), as well as the purpose for accessing the data. NOTE: The way to declare a purpose when accessing the CAMARA APIs is defined in [the CAMARA Security and Interoperability Profile](CAMARA-Security-Interoperability.md#purpose-as-a-scope).
```
Comment (jpengar) [inline documentation/CAMARA-API-access-and-user-consent.md:140]: Do we need to change the link in line 172 as well?
```
This flow is then applicable to On-net scenarios where the mobile connection of the device needs to be authenticated e.g. This flow is for example the one specified for the [CAMARA Number Verification API](https://github.com/camaraproject/NumberVerification/blob/main/documentation/API_documentation/assets/uml_v0.3.jpg) due to the nature of its functionality where a given MSISDN needs to be compared to the MSISDN associated with the mobile connection of the user device.
```

File: documentation/CAMARA-API-access-and-user-consent.md [hunk 2]
```diff
@@ -355,7 +355,7 @@ The documentation template below must be used as part of the API documentation i
 ```
 ### Authorization and authentication
 
-[Camara Security and Interoperability Profile](https://github.com/camaraproject/IdentityAndConsentManagement/blob/main/documentation/CAMARA-Security-Interoperability.md) provides details on how a client requests an access token.
+[Camara Security and Interoperability Profile](CAMARA-Security-Interoperability.md) provides details on how a client requests an access token.
```
Comment (jpengar) [inline documentation/CAMARA-API-access-and-user-consent.md:358]: This is the one being changed in #186

**Review by jpengar — COMMENTED** [review]
Looks good, just a few comments...

File: documentation/CAMARA-API-access-and-user-consent.md (hunk 1 above)
Comment (hdamker) [reply documentation/CAMARA-API-access-and-user-consent.md:140]: Would like too, but it is not a local link. I would ignore it for this release.

**Review by sebdewet — DISMISSED** [review]
LGTM

**Review by AxelNennker — DISMISSED** [review]

File: documentation/CAMARA-API-access-and-user-consent.md (hunk 2 above)
Comment (jpengar) [reply documentation/CAMARA-API-access-and-user-consent.md:358]: But what I mean is #186 and #188 are changing the same document line in different ways...

Comment (hdamker) [reply documentation/CAMARA-API-access-and-user-consent.md]: Ah, yes ... that was the evil of search/replace. Fixed in https://github.com/camaraproject/IdentityAndConsentManagement/pull/188/commits/a4bcb0c67684e289010441cb7847362c35051589

**Review by jpengar — APPROVED** [review]
LGTM

**Pull request merged**